app/components/course-overview.js

```diff
@@ -17,10 +17,16 @@ const Validations = buildValidations({
     }),
   ],
   startDate: [
-    validator('presence', true),
+    validator('date', {
+      dependentKeys: ['model.endDate'],
+      before: reads('model.endDate'),
```

Comment:
I'm thinking this should be onOrBefore to allow for courses to start / end on the same day.